rest_framework/__init__.py

```diff
@@ -6,6 +6,7 @@
 | |\ \| |___/\__/ / | |   | | | | | (_| | | | | | |  __/\ V  V / (_) | |  |   <
 \_| \_\____/\____/  \_/   |_| |_|  \__,_|_| |_| |_|\___| \_/\_/ \___/|_|  |_|\_|
 """
+from .checks import *  # NOQA
```

Comment:
The docs on [Registering and labeling checks](https://docs.djangoproject.com/en/1.11/topics/checks/#registering-and-labeling-checks) suggest using an `AppConfig.ready()`. I think we should follow that. 

[Docs on AppConfig configuration](https://docs.djangoproject.com/en/1.11/ref/applications/#for-application-authors)

1. Add the AppConfig class in `rest_framwork/apps.py`.
2. Import check in `ready`, with comment explaining purpose. 
3. Set `default_app_config` here.